### ISSUE TITLE
Consider renaming Dont* types

### ISSUE THREAD

**Issue by glen-84** [benmccallum/fairybread#82]
I _don't_ (😉) love the use of contractions in type/variable names. What do you think about alternatives like:

`DontValidateAttribute` -> `DisableValidationAttribute` -> f.e. `[DisableValidation]`
`DontValidateImplicitlyAttribute` -> `DisableImplicitValidationAttribute` -> f.e. `[DisableImplicitValidation]`

?

**Comment by benmccallum** [comment]
Yea it did feel a bit dirty when I did it, but I needed to get it out :P 

I'd be happy to take a PR if you're up for it? `[Obsolete]`ing the current one and creating a new one, copy pasting is fine, 

What about `[NoValidation]` to go even shorter?

**Comment by glen-84** [comment]
Hmm.

The only consideration would be the naming of the corresponding fluent API, and whether or not you want those methods to all be verbs.

```c#
.Argument("foo").ValidateWith<FooValidator>(); // verb
.Argument("foo").NoValidation(); // not a verb.
.Argument("foo").NoImplicitValidation(); // not a verb.
```

vs

```c#
.Argument("foo").ValidateWith<FooValidator>(); // verb
.Argument("foo").DisableValidation(); // verb.
.Argument("foo").DisableImplicitValidation(); // verb.
```

I think I prefer the latter, but it's up to you.

**Comment by benmccallum** [comment]
Good point. I think Disable would be the way to go. Since its opt-in by default that makes sense to "disable" the default.
